FirebaseRemoteConfig/Tests/Sample/RemoteConfigSampleApp/ViewController.m

```diff
@@ -0,0 +1,501 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "ViewController.h"
+
+#import <FirebaseAnalytics/FirebaseAnalytics.h>
+#import <FirebaseCore/FIROptions.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseInstanceID/FIRInstanceID+Private.h>
+#import <FirebaseRemoteConfig/FIRRemoteConfig_Private.h>
+#import <FirebaseRemoteConfig/FirebaseRemoteConfig.h>
+#import "FRCLog.h"
+
+static NSString *const FIRPerfNamespace = @"fireperf";
+static NSString *const FIRDefaultFIRAppName = @"__FIRAPP_DEFAULT";
+static NSString *const FIRSecondFIRAppName = @"secondFIRApp";
+
+@interface ViewController ()
+@property(nonatomic, strong) IBOutlet UIButton *fetchButton;
```

Comment:
nit: IBOutlets are generally weak since the view already holds a strong reference to the outlet. For eg. fetchButton is already attached with the view strongly. So fetchButton could as well be weak. Does not make a whole lot of difference. But checking if I'm missing any scenario here.

Reply:
Ack. Am migrating the existing (internal) app for this CL. Will have a cleanup CL and use this input then.